hibernate-reactive-core/src/test/java/org/hibernate/reactive/AutoincrementTest.java

```diff
@@ -178,21 +171,17 @@ public int hashCode() {
 		Double nullDouble;
 		Byte nullByte;
 
-		BigDecimal decimal;
-		BigInteger integer;
 		byte[] bytes;
 		Cover cover;
 		@javax.persistence.Basic
 		Serializable thing;
 		TimeZone timeZone;
 		@Temporal(TemporalType.DATE)
 		private Date date;
-		@Column(name="_localDate")
+		@Column(name="alocalDate")
```

Comment:
This change is needed because DB2 does not support column names that start with an underscore

Reply:
> Can this be done in ORM?

No, ORM doesn't automatically quote or change what the user defines explicitly via `@Column` - we discussed that and it's really not simple.